web/src/routes/__tests__/SearchPage.spec.tsx

```diff
@@ -7,10 +7,10 @@ import { Route } from 'react-router-dom'
 import { ThemeProvider } from 'styled-components'
 import LanguageModelBuilder from '../../../../api-client/src/testing/LanguageModelBuilder'
 import buildConfig from '../../constants/buildConfig'
-import { mockUseLoadFromEndpointWitData } from '../../testing/mockUseLoadFromEndpoint'
 import { renderWithBrowserRouter } from '../../testing/render'
 import { createPath, RoutePatterns } from '../index'
 import SearchPage from '../SearchPage'
+import { mockUseLoadFromEndpointWitData } from '../../../../api-client/src/testing/mockUseLoadFromEndpoint'
```

Comment:
```suggestion
import { mockUseLoadFromEndpointWitData } from 'api-client/src/testing/mockUseLoadFromEndpoint'
```

Reply:
I also wondered about that already, not sure :/

Reply:
gonna be resolved in separate issue :)